e2e-tests/cypress/tests/cypress/e2e/models/UserEMailSite.ts

```diff
@@ -0,0 +1,17 @@
+/// <reference types="cypress" />
+
+export class UserEMailSite {
+  // selectors
+  emailInbox = ".sidebar-emails-container";
+  emailList = ".email-list";
+  emailMeta = ".email-meta";
+  emailSubject = ".subject";
+
```

Comment:
Please replace " by ' and remove semicolon ;

Reply:
There is a separate issue for the linting issues